### ISSUE TITLE
Add Search query to URL

### ISSUE THREAD

**Issue by mmmavis** [mozilla/network-pulse#114]
From @kristinashu in https://github.com/mozilla/network-pulse/issues/58#issuecomment-235982743

> If I enter a search term, I should be able to copy that url and send it to someone so they go straight to a page with those search results (and any new ones that my have been posted during fitting the criteria). This is something that came up in an interview today and would be very useful.

Depends on #58, #99

**Comment by xmatthewx** [comment]
duplicate of #67
